src/Main.scala

```diff
@@ -0,0 +1,14 @@
+/**
+  * Created by jakefroeb on 10/28/16.
+  */
+object Main {
+  def main(args: Array[String]): Unit = {
+    val cloud = Cloud("Cumulus",12,14,23)
+    cloud.printCloud
+    val dog = Dog("Chopper", "Golden", 16)
+    dog.printDog
```

Comment:
fyi, interestingly, when you are only going to use an object once then simply do this for the same effect:
  Cloud("Cumulus",32,3,9).printCloud
  Dog("Jet", "Mutt", 11).printDog

What's really cool is you can remove the .printCloud and .printDog  and just do a println() instead of "def printDog = println()" in the case class. So, it'll look like this in Main:
  Cloud("Cumulus",32,3,9)
  Dog("Jet", "Mutt", 11)